e2e-test/cbas_azure_e2e_test.py

```diff
@@ -0,0 +1,219 @@
+from workspace_helper import create_workspace
+from app_helper import create_app, poll_for_app_url
+import requests
+import os
+import json
+import random
+import string
+import uuid
+import time
+
+
+# Setup configuration
+# These values should be injected into the environment before setup
+azure_token = os.environ.get("AZURE_TOKEN")
+bee_name = os.environ.get("BEE_NAME")
+billing_project_name = os.environ.get("BILLING_PROJECT_NAME")
+
+rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+
+
+# Upload data to WDS
+def upload_wds_data(wds_url, workspace_id, tsv_file_name, record_name):
+    #open TSV file in read mode
+    tsv_file = open(tsv_file_name, "r")
+    request_file = tsv_file.read()
+    tsv_file.close()
+
+    uri = f"{wds_url}/{workspace_id}/tsv/v0.2/{record_name}"
+    headers = {"Authorization": f"Bearer {azure_token}"}
+
+    response = requests.post(uri, files={'records':request_file}, headers=headers)
+
+    status_code = response.status_code
+
+    if status_code != 200:
+        print(response.text)
+        exit(1)
+    print(f"Successfully uploaded data to WDS. Response: {response.json()}")
+
+
+# Create no-tasks-workflow method in CBAS
+def create_cbas_method(cbas_url, workspace_id):
+    method_name = "no-tasks-workflow"
```

Comment:
These methods could potentially be pulled out into another helper file

Reply:
I thought about that, but I wasn't sure if any other test apart from CBAS E2E test might use this as these methods are creating and submitting a specific `no-tasks-workflow.wdl` to WORKFLOWS app instead of being general purpose. But if reviewers have strong opinion then I can pull these methods into `workflows_helper` file.

Reply:
Not a very strong opinion.  I suppose if anyone ever needs them later, they can refactor them out themselves.